Foundation/NSGeometry.swift

```diff
@@ -36,7 +37,7 @@ public struct CGFloat {
 extension CGFloat: Comparable { }
 
 public func ==(lhs: CGFloat, rhs: CGFloat) -> Bool {
-    return lhs.native == rhs.native
+    return abs(lhs.native - rhs.native) < DBL_EPSILON
```

Comment:
albeit that this is probably what most people mean:
the definition on darwin is
```swift
public func ==(lhs: CGFloat, rhs: CGFloat) -> Bool {
  return lhs.value == rhs.value
}
```
see https://github.com/apple/swift/blob/master/stdlib/public/SDK/CoreGraphics/CoreGraphics.swift